receptorctl/test-requirements.txt

```diff
@@ -1,6 +1,8 @@
 -r build-requirements.txt
-pytest==6.1.1
+-e .
```

Comment:
I'm not sure why you needed to add this line but the tests work fine for me when deleting it and it also fixes the container build.

Reply:
I think you already told me to not do that. I didn't realize that this file was being read.
Thank you for investigating that!